src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1338,6 +1297,39 @@ internal static string GetModulePath()
             return currentModulePath;
         }
 
+#if !UNIX
+        /// <summary>
+        /// Returns a PSModulePath suiteable for Windows PowerShell by removing this PowerShell's specific
+        /// paths from current PSModulePath.
+        /// </summary>
+        /// <returns>
+        /// Returns appropriate PSModulePath for Windows PowerShell.
+        /// </returns>
+        internal static string GetWindowsPowerShellModulePath()
+        {
+            string currentModulePath = GetModulePath();
+
+            if (currentModulePath != null)
```

Comment:
If you invert this condition and return `null` early, you can put the important part of this method in the main body rather than under an if clause

Reply:
will update